app/models/orphan.rb

```diff
@@ -124,6 +124,10 @@ def current_sponsor
     current_sponsorship.sponsor if currently_sponsored?
   end
 
+  def resolve_sponsorship_status
+    reactivate and save!
```

Comment:
`#reactivate` has all the necessary functionality, but its name is obviously ill-suited here